src/test/resources/org/prebid/server/it/openrtb2/kidoz/test-cache-kidoz-response.json

```diff
@@ -0,0 +1,13 @@
+{
+  "responses": [
+    {
+      "uuid": "f0ab9105-cb21-4e59-b433-70f5ad6671cb"
+    },
+    {
+      "uuid": "e7965b2e-0aa3-4252-a22c-580ed010e619"
+    },
+    {
+      "uuid": "44a52b06-b29f-4819-a05f-db36b9e7b8fc"
+    }
```

Comment:
do you really need third cache response ?

Reply:
Yes, because we have video bid that can cache vastxml:
"vastXml": {
                  "url": "{{ cache.resource_url }}44a52b06-b29f-4819-a05f-db36b9e7b8fc",
                  "cacheId": "44a52b06-b29f-4819-a05f-db36b9e7b8fc"
                }